setup.py

```diff
@@ -51,7 +51,7 @@
 
 pandas_requirements = [
     # Must be kept in sync with pyproject.toml
-    "pyarrow>=3.0.0,<3.1.0",
+    "pyarrow>=3.0.0,<4.1.0",
```

Comment:
Arrow has no ABI stability thus the following is better suited for wheels.
```suggestion
    "pyarrow>=4.0.0,<4.1.0",
```

Reply:
Thank you for this! ❤️